src/main/java/com/google/sps/servlets/UserLoginServlet.java

```diff
@@ -34,11 +37,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String loggedOutUrl = userService.createLogoutURL("/index.html");
       loginInfo.addProperty("LogOutUrl", loggedOutUrl);
       loginInfo.addProperty("LogInUrl", "");
-
+      try {
+        if(SessionContext.getInstance() != null){
```

Comment:
This if statement can be removed (L41) since getInstance will not return null.

Reply:
Done